src/test/java/org/folio/moduserstest/ProxyRelationshipsIT.java

```diff
@@ -64,8 +80,10 @@ public static void beforeAll(Vertx vertx, VertxTestContext context) {
   }
 
   @BeforeEach
-  public void beforeEach() {
+  public void beforeEach(VertxTestContext context) {
     proxiesClient.deleteAllProxies();
+
+    context.completeNow();
```

Comment:
since there is only synchronous code in this block I dont see the reason why test context should be used here

Reply:
removed